arch/arc/Kconfig

```diff
@@ -38,6 +38,8 @@ config ARC
 	select HAVE_KERNEL_LZMA
 	select HAVE_KPROBES
 	select HAVE_KRETPROBES
+	select HAVE_FUNCTION_TRACER
```

Comment:
Since for now it only reliably works for ARCv3, don't we want to move it to the corresponding section to not tempt users to play with a broken thing?

Reply:
Thank you for your comment! In that case the build would fail due to the Makefile, but its a lot cleaner to add a check for ISA_ARCV3 directly in the options as you suggest!